packages/ckeditor5-emoji/docs/api/emoji.md

```diff
@@ -2,11 +2,11 @@
 category: api-reference
 ---
 
-# Emoji feature for CKEditor&nbsp;5
+# CKEditor&nbsp;5 Emoji feature
 
 [![npm version](https://badge.fury.io/js/%40ckeditor%2Fckeditor5-emoji.svg)](https://www.npmjs.com/package/@ckeditor/ckeditor5-emoji)
 
-This package implements the emoji feature for CKEditor&nbsp;5.
+This package implements the Emoji feature for CKEditor&nbsp;5.
```

Comment:
Yeah, searching for "emoji feature" (lower case) returns a few other places to update I guess.